client/knowledgebase/components/App.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+import { KnowledgeBase, Launcher } from '../containers';
+
+function App(props, dispatchProps) {
+  console.log('props: ', props);
+  console.log('dispatchProps: ', dispatchProps);
+  console.log('loadType: ', props.data.kbLoader.loadType);
+
+  const loadType = props.data.kbLoader.loadType;
+  const widgetClasses = classNames('erxes-widget-kb');
+
+  if (loadType === 'embedded') {
+    return (
+      <div className={widgetClasses}>
+        <KnowledgeBase />
+      </div>
+    );
+  } else if (loadType === 'shoutbox') {
```

Comment:
Don't need to write `else` keyword if above statement is returning.

Reply:
okay, but its kinda debatable